app/src/pages/PageJoinNetwork.re

```diff
@@ -0,0 +1,52 @@
+open Atom;
```

Comment:
The good thing about our pages so far was that they didn't assume anything about the outer structure they were rendered into and I think we should preserve this property. So maybe we can extract the button component which fires up the modal and keep the join network page purely concerned with the form and logic of capturing the data we like to get from a user.

Reply:
Also good point, I took it verbatim from our DAPP exploration. Will have a look on how to do it better.